programs/clearing_house/src/math/amm.rs

```diff
@@ -323,17 +326,32 @@ pub fn update_mark_twap(
         "amm.last_oracle_price <= 0"
     )?;
 
-    // optimistically estimation of bid/ask using execution premium
+    // estimation of bid/ask by looking at execution premium
```

Comment:
might be good to add a rust test or two to make sure the bid/ask twaps get updated as you would expect